### ISSUE TITLE
Research Style Guide

### ISSUE THREAD

**Issue by kellysmith1008** [usds/ditap-curriculum-update#85]
**Information**
- This ticket is to review the current voice, tone, and style of DITAP material. What's the voice? What's the tone? Do we use second person or third person point of view? Are we friendly or formal in our materials? What is our heading structure? Do we always use DITAP in all caps, for example, or are there times we don't? Are there DITAP-specific language, acronyms, or things we need to document? What are our best practices around creating deliverables? Do we always include alt text in images or are some decorative images only? Should we include images? Do we use sentence-case or title-caseMore information is in the tasks, but that's the idea behind this ticket. We need to determine what users are expecting first and compare/align those expectations with industry standards and 508 compliance. This will allow us to build the style guide for the 40% secret sauce.

**Tasks**
- [x] Review DITAP material and document voice and tone
- [x] Review material and document DITAP-specific guidance (like always capitalize DITAP, for example)
- [x] Review material and document heading structures, typography, and title structure for readings, slides, etc
- [x] Review material and document video settings - do they include a transcript? Are they embedded or do they link out to the video?
- [x] Review material and document links - are they internal or external? If they are external links, do they include a visual and CSS cue? 

**Definition of success**
- [x] DITAP deliverables have been reviewed for voice and tone
- [x] DITAP-specific language has been documented
- [x] Deliverables have been reviewed for headings and typography
- [x] Discrepancies (in voice, tone, language, etc) have been documented 

**Other information**
- There will be a follow-up ticket once this research is done to start building out the DITAP style guide. This ticket is solely around doing the research to determine what (or if we have) the current voice and tone is for the program and if it needs to also be refreshed

**Comment by kellysmith1008** [comment]
[Document used for discovery and research](https://docs.google.com/document/d/1dzodHvFW_iIH2PxSttrPw60lPr12Loshno1w4puo3Co/edit?tab=t.0)
